### DISCUSSION TITLE
Can we update the rank of edge?

### DISCUSSION THREAD

**Discussion by akhilravuri-tul-scm** [vesoft-inc/nebula#4323] [Q&A] [answered]
Hi,

I saw documentation about updating edge but I am looking of updating the rank of edge.

I just need to update the rank no need to change the data.

Thanks,
Akhil

**Comment by wey-gu** [comment, marked as answer]
Sorry, the rank is part of the `4-tuple` or actually the `id` to distinguish an instance of edge, thus changing `rank` for a given edge isn't updating anymore, think of `updating` of vertex ID of a vertex isn't a thing.
